stellar/its.js

```diff
@@ -90,6 +111,9 @@ async function deployRemoteInterchainToken(wallet, _, chain, contract, args, opt
     const gasTokenAddress = options.gasTokenAddress || chain.tokenAddress;
     const gasAmount = options.gasAmount;
 
+    printInfo('Original salt', salt);
```

Comment:
```suggestion
    printInfo('Salt', salt);
```
name it after the input maybe

Reply:
make sense, updated.